src/main/java/core/basesyntax/service/reader/ReaderServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.reader;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReaderServiceImpl implements ReaderService {
+    @Override
+    public List<String> readFile(String data) {
+        Path path = Paths.get(data);
```

Comment:
no need in this line as you use it only once insert it directly where you use it

Reply:
agree, fixed